app/src/Rating/Rating.tsx

```diff
@@ -111,15 +115,15 @@ export const ReviewHolder: React.FC<{classNum : string}> = ({classNum}) => {
                 </Popup>
             }
             <div className="scales">
-                <RatingScale category={1} setReview={setRatingContents}/>
-                <RatingScale category={2} setReview={setRatingContents}/>
-                <RatingScale category={3} setReview={setRatingContents}/>
+                <RatingScale category={1} setReview={setRatingContents} initialValue={initialState.rating_one}/>
+                <RatingScale category={2} setReview={setRatingContents} initialValue={initialState.rating_two}/>
+                <RatingScale category={3} setReview={setRatingContents} initialValue={initialState.rating_three}/>
             </div>
             <div className="other-inputs">
-                <InputField setReview={setRatingContents} field="Quarter Taken"/>
-                <InputField setReview={setRatingContents} field="Professor"/>
+                <InputField setReview={setRatingContents} initialValue={reviewState !== undefined ? initialState.quarter : ""} field="Quarter Taken"/>
```

Comment:
I would put these parameters on separate lines to improve readability

Reply:
Fixed